libs/blocks/list.js

```diff
@@ -8,6 +8,7 @@ const blocks = [
   'how-to',
   'modal',
   'marquee',
+  'review',
```

Comment:
Changes to list.js need to go into their own separate PR.  This is to ensure that enabling a milo block doesn't adversely affect consumers.

Reply:
will make the changes accordingly. Thanks!!

Reply:
Done!!